ImperatorToCK3/Outputter/BookmarkOutputter.cs

```diff
@@ -1,30 +1,51 @@
 ﻿using commonItems;
+using ImageMagick;
 using ImperatorToCK3.CK3.Characters;
+using ImperatorToCK3.CK3.Map;
 using ImperatorToCK3.CK3.Titles;
+using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Text;
 
 namespace ImperatorToCK3.Outputter {
 	public static class BookmarkOutputter {
-		public static void OutputBookmark(string outputModName, Dictionary<string, Character> characters, Dictionary<string, Title> titles, Date ck3BookmarkDate) {
-			var path = "output/" + outputModName + "/common/bookmarks/00_bookmarks.txt";
+		public static void OutputBookmark(Dictionary<string, Character> characters, Dictionary<string, Title> titles, Configuration config) {
+			OpenCL.IsEnabled = true; // enable OpenCL in ImageMagick
+			var path = "output/" + config.OutputModName + "/common/bookmarks/00_bookmarks.txt";
 			using var stream = File.OpenWrite(path);
 			using var output = new StreamWriter(stream, Encoding.UTF8);
 
+			// get province positions
```

Comment:
On a design level, it probably makes more sense to not input things in an outputter. Have CK3 world import them and pass them in here. You'll probably find uses for them elsewhere anyways.